runbotics/runbotics-orchestrator-ui/src/views/action/ActionDetails/index.tsx

```diff
@@ -20,31 +9,34 @@ import { useSnackbar } from 'notistack';
 import { IAction } from 'src/types/model/action.model';
 import { saveAction, setShowEditModal } from 'src/store/slices/Action/Action.thunks';
 import { useDispatch, useSelector } from 'src/store';
-import JSONSchemaFormRenderer from
-    'src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/JSONSchemaFormRenderer';
+import JSONSchemaFormRenderer from 'src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/JSONSchemaFormRenderer';
 import customWidgets from 'src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets';
 import ErrorBoundary from 'src/components/utils/ErrorBoundary';
-import useTranslations, { translate as t } from 'src/hooks/useTranslations';
+import useTranslations from 'src/hooks/useTranslations';
 
 const Form = withTheme<any>(Mui5Theme) as FC<FormProps<any> & { ref: any }>;
 
-const schema: JSONSchema7 = {
-    type: 'object',
-    properties: {
-        script: {
-            type: 'string',
-            title: t('Action.Details.Script'),
-        },
-        label: {
-            type: 'string',
-            title: t('Action.Details.Label'),
-        },
-        form: {
-            type: 'string',
-            title: t('Action.Details.Form'),
+const useSchema = (): JSONSchema7 => {
```

Comment:
to another file and include information about action details in the name like `useActionDetailsSchema`